eng/projlist/Azure.Core_dependencylist.txt

```diff
@@ -0,0 +1,381 @@
+$(RepoRoot)sdk\advisor\Azure.ResourceManager.Advisor\tests\Azure.ResourceManager.Advisor.Tests.csproj
```

Comment:
Did you intend to include this file or is it supposed to be generated?

Reply:
This is intended to be removed. I somehow included it again.